client/libra_wallet/src/mnemonic.rs

```diff
@@ -20,7 +20,22 @@ use std::{
 #[cfg(test)]
 use tempfile::NamedTempFile;
 
-/// Mnemonic seed for deterministic key derivation
```

Comment:
You may want to convey the intent w.r.t BIP-39 by pointing to https://github.com/bitcoin/bips/wiki/Comments:BIP-0039

Reply:
This is still active :-/